src/dash-table/components/Table/index.tsx

```diff
@@ -57,7 +58,15 @@ export default class Table extends Component<PropsWithDefaultsAndDerived, Standa
         this.controlled = this.getControlledProps();
         this.updateDerivedProps();
 
-        return (<ControlledTable {...this.controlled} />);
+        return (<TooltipCursorDecorator
+            column_static_tooltip={this.controlled.column_static_tooltip}
+            column_conditional_tooltips={this.controlled.column_conditional_tooltips}
+            tooltips={this.controlled.tooltips}
+            delay={500}
+            virtualized={this.controlled.virtualized}
+        >
+            <ControlledTable {...this.controlled} />
+        </TooltipCursorDecorator>);
```

Comment:
The tooltip decorator (a) shows the tooltip, (b) wraps up whatever is under its sphere of influence (e.g the ControlledTable) -- the table is unaware that it is decorated with tooltips.